src/fields/TextElement.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Delta from 'quill-delta';
+import generateNodes from './textNodes';
+
+const buttonAlignmentMap = {
```

Comment:
this should only be defined one time between text and buttonelement

Reply:
extracted this to utils.